generate_compose/odcs_compose_generator.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+"""Initialize and use a compose generator using ODCS"""
+
+from pathlib import Path
+
+import click
+import yaml
+
+from .compose_generator import ComposeGenerator
+from .odcs_configurations_generator import ODCSConfigurationsGenerator
+from .odcs_fetcher import ODCSFetcher
+from .odcs_requester import ODCSRequester
+
+
+@click.command()
+@click.option(
+    "--compose-path",
+    help="Path to where the compose should be saved.",
```

Comment:
Gonna be the path to the compose file itself

Reply:
done